corsheaders/models.py

```diff
@@ -3,3 +3,7 @@
 
 class CorsModel(models.Model):
     cors = models.CharField(max_length=255)
+
+    @python2_unicode_compatible
```

Comment:
(you need to add `from django.utils.encoding import python_2_unicode_compatible`)

Also it's `python_2` not `python2`

Also it needs to apply to the class not the method!